src/test/java/teammates/test/cases/browsertests/AdminHomePageUiTest.java

```diff
@@ -212,49 +213,184 @@ private void testCreateInstructorAction() throws Exception {
         ______TS("new instructor can access sample course students page");
         coursesPage.loadStudentsTab().verifyHtmlMainContent("/newlyJoinedInstructorStudentListPage.html");
 
-        ______TS("new instructor can view feedbackSession result of sample course");
+        ______TS("new instructor can view result of First team feedback session of sample course");
         coursesPage.loadInstructorHomeTab();
         instructorHomePage = AppPage.getNewPageInstance(browser, InstructorHomePage.class);
-        instructorHomePage.clickFeedbackSessionViewResultsLink(demoCourseId, "Second team feedback session");
-        instructorHomePage.verifyHtmlMainContent("/newlyJoinedInstructorFeedbackResultsPage.html");
+        InstructorFeedbackResultsPage resultsPage =
+                instructorHomePage.clickFeedbackSessionViewResultsLink(demoCourseId, "First team feedback session");
+        if (resultsPage.isElementPresent("collapse-panels-button")) {
+            resultsPage.clickCollapseExpandButton();
+            resultsPage.waitForPanelsToExpand();
+        }
+        instructorHomePage.verifyHtmlMainContent("/newlyJoinedInstructorFirstFeedbackSessionResultsPage.html");
+
+        ______TS("new instructor can view result of Second team feedback session of sample course");
+        coursesPage.loadInstructorHomeTab();
+        instructorHomePage = AppPage.getNewPageInstance(browser, InstructorHomePage.class);
+        resultsPage =
+                instructorHomePage.clickFeedbackSessionViewResultsLink(demoCourseId, "Second team feedback session");
+        if (resultsPage.isElementPresent("collapse-panels-button")) {
+            resultsPage.clickCollapseExpandButton();
+            resultsPage.waitForPanelsToExpand();
+        }
+        instructorHomePage.verifyHtmlMainContent("/newlyJoinedInstructorSecondFeedbackSessionResultsPage.html");
+
+        ______TS("new instructor can view result of Third team feedback session of sample course");
+        coursesPage.loadInstructorHomeTab();
+        instructorHomePage = AppPage.getNewPageInstance(browser, InstructorHomePage.class);
+        resultsPage = instructorHomePage.clickFeedbackSessionViewResultsLink(
+                demoCourseId, "Session with different question types");
+        if (resultsPage.isElementPresent("collapse-panels-button")) {
+            resultsPage.clickCollapseExpandButton();
+            resultsPage.waitForPanelsToExpand();
+        }
+        instructorHomePage.verifyHtmlMainContent("/newlyJoinedInstructorThirdFeedbackSessionResultsPage.html");
```

Comment:
This whole block 239-247 is repeated multiple times. May be extract into something like this?
```java
verifySessionResults(demoCourseId, "Session with different question types",
        "/newlyJoinedInstructorThirdFeedbackSessionResultsPage.html");
```